src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, sex, slug } = person;
+  const FEMALE = 'f';
```

Comment:
do you really need to create this variable on each iteration? 
let's move it outside of component